src/main/java/org/terasology/taskSystem/actions/PerformTaskNode.java

```diff
@@ -96,6 +96,20 @@ private void changeOreonAttributes(Actor oreon, TaskComponent taskComponent) {
                 oreonAttributeComponent.hunger = 0;
                 break;
 
+            case AssignedTaskType.Train_Strength:
+                oreonAttributeComponent.strength += 10;
+                oreonAttributeComponent.strength = oreonAttributeComponent.strength > oreonAttributeComponent.maxStrength
```

Comment:
Assigning the same value doesn't make sense.  Just use an if ( str > max) { str = max; } statement.